### ISSUE TITLE
Color coding the bed track

### ISSUE THREAD

**Issue by babakala** [paul-shannon/igvShiny#42]
HI @paul-shannon,

Wondering if one could specify the itemRGB column for the bed file to be able to color code different elements?

**Comment by paul-shannon** [comment]
@babakala - can you provide a short example bedfile?

**Comment by babakala** [comment]
Sure, here it is:

```
track itemRgb=On
chr2    1000    2000    green 0   -   1000    2000    0,255,128
chr2    1010    2020    blue 0   +   1010    2020    0,128,255
```
![Screenshot 2023-02-07 at 21 44 27](https://user-images.githubusercontent.com/91609749/217361554-73dad184-f252-4100-b177-05d662b4658f.png)

**Comment by paul-shannon** [comment]
I’ll get to this in the next few days.  Thanks for the request.

 - Paul

**Comment by babakala** [comment]
Thanks Paul, I think it would also be useful if one could visualise all the other columns from a bed file as well e.g. thickStart and thickEnd. see here: https://genome.ucsc.edu/FAQ/FAQformat.html#format1

**Comment by paul-shannon** [comment]
@babakala - I am only getting around to this now, sorry for my delay.
In addition to the variety of forms your link describes (3 required columns, nine optional ones) 
I see that this link,  https://bedtools.readthedocs.io/en/latest/content/general-usage.html,
identifies BED[3,4,5,6,12] formats.
   
It would be a big help to me if you could, with your vested interest :) simplify my job by curating an example of each and all of these.  I mostly work on other projects these days, but would like to get this right.  And do so by coming up with a comprehensive solution, so I won't need to revisit this in the future.
Could you contribute a good example of each and every possible bed format?  Perhaps as a PR?  I'd be grateful.

**Comment by paul-shannon** [comment]
@babakala I tracked the problem down to igv.js, reported in https://github.com/igvteam/igv.js/issues/1597
Jim Robinson is usually pretty quick in making the fix.
![image](https://user-images.githubusercontent.com/2480712/220232654-577d9b46-7f91-48eb-8bb6-6982e9d9740f.png)

**Comment by paul-shannon** [comment]
fixed in igvShiny 1.5.9009
<img width="981" alt="image" src="https://user-images.githubusercontent.com/2480712/220354701-555587ef-56a0-44e2-b132-3ab6a007e260.png">
